Tests/Unit/Microsoft365DSC/Microsoft365DSC.EXOReportSubmissionPolicy.Tests.ps1

```diff
@@ -0,0 +1,267 @@
+        Invoke-Command -ScriptBlock $Global:DscHelper.InitializeScript -NoNewScope
+
+        BeforeAll {
+            $secpasswd = ConvertTo-SecureString 'test@password1' -AsPlainText -Force
+            $Credential = New-Object System.Management.Automation.PSCredential ('tenantadmin@mydomain.com', $secpasswd)
+
+            Mock -CommandName Confirm-M365DSCDependencies -MockWith {
+            }
+
+            Mock -CommandName New-M365DSCConnection -MockWith {
+                return 'Credentials'
+            }
+
+            Mock -CommandName Get-PSSession -MockWith {
+            }
+
+            Mock -CommandName Remove-PSSession -MockWith {
+            }
+
+            Mock -CommandName New-ReportSubmissionPolicy -MockWith {
+            }
+
+            Mock -CommandName Set-ReportSubmissionPolicy -MockWith {
+            }
+
+            Mock -CommandName Remove-ReportSubmissionPolicy -MockWith {
+            }
+
+            # Mock Write-Host to hide output during the tests
+            Mock -CommandName Write-Host -MockWith {
+            }
+
+            Mock -CommandName Write-Warning -MockWith {
+            }
+        }
+
+        # Test contexts
+        Context -Name 'ReportSubmissionPolicy creation.' -Fixture {
+            BeforeAll {
+                $testParams = @{
+                    Ensure                                 = 'Present'
+                    Credential                             = $Credential
+                    IsSingleInstance                       = 'Yes';
+                    Identity                               = "DefaultReportSubmissionPolicy"
+                    DisableQuarantineReportingOption       = $False;
+                    EnableCustomNotificationSender         = $False;
+                    EnableOrganizationBranding             = $False;
+                    EnableReportToMicrosoft                = $True;
+                    EnableThirdPartyAddress                = $False;
+                    EnableUserEmailNotification            = $False;
+                    PostSubmitMessageEnabled               = $True;
+                    PreSubmitMessageEnabled                = $True;
+                    ReportJunkToCustomizedAddress          = $False;
+                    ReportNotJunkToCustomizedAddress       = $False;
+                    ReportPhishToCustomizedAddress         = $False;
+                }
+
+
+                Mock -CommandName Get-ReportSubmissionPolicy -MockWith {
+                    return @{
+
+                    }
+                }
+            }
+
+            It 'Should return Absent from the Get method' {
+                (Get-TargetResource @testParams).Ensure | Should -Be 'Absent'
+            }
+
+            It 'Should return false from the Test method' {
+                Test-TargetResource @testParams | Should -Be $false
+            }
+
+            It 'Should call the Set method' {
+                Set-TargetResource @testParams
```

Comment:
Same as the other comment. How should Pester validate that the function ran successfully. In this case the Invoke New-ReportSubmissionPolicy and Set-ReportSubmissionPolicy sounds logical.